### DISCUSSION TITLE
ThermoMechanical Simulation

### DISCUSSION THREAD

**Discussion by deepakbiitk** [peridigm/peridigm#377] [General]
Hi all,

I am exploring solution to thermo-mechanical problems with Peridigm. I solved the diffusion equation to obtain temperature distribution. Now I want to use the temperature field obtained to PD mechanical problem. In the example files the prescribed temperature uses temperature functions. I am not sure, how do I transfer the temperature data to obtain displacement solution. In the talk on Peridigm in One Nonlocal World, it was mentioned Peridigm does have the one way coupling capability for multiphysics solution. Can anybody please guide how can I set up the input file.

I appreciate your help.

Best,
Deepak

**Comment by djlittl** [comment]
See the test problem Notched_Plate_Data_Loader for one-way coupling.

**Comment by JTHesse** [comment]
Take a look at [PeriLab.jl](https://github.com/PeriHub/PeriLab.jl). Thermo-mechanical is implemented and verified there.
